### ISSUE TITLE
The code for implimentation of ELPH and NCNC on non-attributed benchmarks.

### ISSUE THREAD

**Issue by junwon0** [Barcavin/efficient-node-labelling#7]
Can you share the code and the hyperparameter what you use for the experiments?

Thanks,
Junwon

**Comment by Barcavin** [comment]
Please find the logs about the hyperparameter of ELPH and NCNC attached below.

What we did was grid-search the hyperparameters that have been tuned in ELPH and NCNC in their repos. Then we picked the one with best validation accuracy.

Let me know if you have more questions.

Best,
Kaiwen

[ELPH_log.txt](https://github.com/user-attachments/files/18432816/ELPH_log.txt)
[NCNC_log.txt](https://github.com/user-attachments/files/18432817/NCNC_log.txt)
